samples/yesno/devops_pipelines/yesno-ci.yml

```diff
@@ -0,0 +1,56 @@
+trigger:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - samples/yesno/ml_model
+    - samples/yesno/ml_service
+
+variables:
+- template: yesno-variables-template.yml
+- group: iac-aml-vg
```

Comment:
How about renaming that Variable Group that it can be independently filled and won't conflict with other examples? Currently it potentially mixes up data needed by the IaC Pipelines and this example. How about `kaldi-yesno-aml-vg`?

Reply:
we are using same variable group as we reuse common/infrastructure to setup environment.